src/cnd/cnd.ts

```diff
@@ -69,6 +69,24 @@ export interface EthereumCallContractPayload {
   network: string;
 }
 
+export interface LndSendPaymentPayload {
+  public_key: string;
+  amount: string;
+  secret_hash: string;
+  final_cltv_delta: number;
+}
+
+export interface LndAddHoldInvoicePayload {
+  amount: string;
+  secret_hash: string;
+  expiry: number;
+  memo: string;
```

Comment:
After further consideration, I believe the user should be the one to decide how to name the COMIT swap invoices and memo should not be part of the action.

@thomaseizinger thoughts?

Reply:
Yeah I am fine with that :)

Reply:
Follow-up PR.